ios-target-app/Networking/Services/AuthenticationServices.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  AuthenticationServices.swift
+//  ios-target-app
+//
+//  Created by Raul Piñeres Carrera on 23/09/22.
+//
+
+import Foundation
+import UIKit
+import RSSwiftNetworking
+
+internal class AuthenticationServices {
+    
+    enum AuthError: Error {
+        case userSessionInvalid
+    }
+    
+    private let apiClient: APIClient
+    
+    init(
+        apiClient: APIClient = iOSBaseAPIClient.shared
+    ) {
```

Comment:
Pls don't split lines if not necessary:
```suggestion
    init(apiClient: APIClient = iOSBaseAPIClient.shared) {
```